README.md

```diff
@@ -57,7 +57,7 @@ The specification and code is licensed under the Apache 2.0 license found in the
 
 ## Code of Conduct
 
-Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](code-of-conduct.md).
+Participation in the OpenContainers community is governed by [OpenContainer's Code of Conduct](https://github.com/opencontainers/tob/blob/master/code-of-conduct.md).
```

Comment:
@vbatts Did you discuss this in 2016-04-20 call?
I guess we should keep it since we need it in PDF?